Source/Scene/Cesium3DTile.js

```diff
@@ -322,22 +313,25 @@ define([
 
         // Members that are updated every frame for tree traversal and rendering optimizations:
         this._distanceToCamera = 0;
-        this._visibilityPlaneMask = 0;
-        this._childrenVisibility = Cesium3DTileChildrenVisibility.VISIBLE;
-        this._lastSelectedFrameNumber = -1;
+        this._centerZDepth = 0; // TODO : remove?
```

Comment:
Just a reminder for the `TODO`.

Reply:
Thanks, TODO removed - we still need that variable.